README.rst

```diff
@@ -473,9 +475,9 @@ The following options (``[flags]``) are supported:
         advertise_uri: localhost:3302
         http_port: 8082
 
-  Parameters that can be specified in ``instances.yml`` are listed
-  `here <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/modules/cartridge/#cfg-opts-box-opts>`_.
-  The ``advertise_uri`` parameter is required.
+  For a full list of parameters that can be specified in ``instances.yml``, read the
+  `cartridge.cfg() documentation <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/modules/cartridge/#cfg-opts-box-opts>`_.
```

Comment:
this should be a `:ref:` link

Reply:
I made this a `:doc:` link according to [the SG](https://www.tarantool.io/en/doc/latest/contributing/docs/markup/links/#linking-to-other-documentation-pages).